### PR TITLE
Fix Google Search errors

### PR DESCRIPTION
**Pull request by sweatC**
Related #322

### CODE REVIEW

**Comment by sweatC** [comment]
To fix this issues we need to specify one of the parameters of our ItemList because google interpreters it as Product that needs to have one of this attributes( "offers", "review", or "aggregateRating"). I don't know what is more important, so I have added aggregateRating with random values. @pftg you need to provide true values for this.
Staging: http://jetthoughts-pr-404.surge.sh/

File: _includes/head.html [hunk 1]
```diff
@@ -112,7 +112,12 @@
           "position": 1,
           "name": "Agent Inbox",
           "description": "Web and mobile development",
-          "url": "{{ site.url }}/agentinbox"
+          "url": "{{ site.url }}/agentinbox",
```
Comment (pftg) [inline _includes/head.html:115]:
```suggestion
          "url": "{{ site.url }}/agentinbox.html",
```

File: _includes/head.html [hunk 2]
```diff
@@ -112,7 +112,12 @@
           "position": 1,
           "name": "Agent Inbox",
           "description": "Web and mobile development",
-          "url": "{{ site.url }}/agentinbox"
+          "url": "{{ site.url }}/agentinbox",
+          "aggregateRating": {
```
Comment (pftg) [inline _includes/head.html:116]: could you check what others do for such cases, give me some info for thoughts. As for me `review` much close.

**Review by pftg — COMMENTED** [review]
Give me how others do this

File: _includes/head.html (hunk 2 above)
Comment (sweatC) [reply _includes/head.html:116]: List with ratings:
image
Offers:
image
Reviews:
image
@pftg for more details see: https://developers.google.com/search/docs/data-types/product

Comment (pftg) [reply _includes/head.html:116]: it's not related to agent inbox, what should we use another for such case?

**Comment by sweatC** [comment]
This task was fixed. 10 Apr the errors were disappeared. @andriyParashchuk said that it was problems with agent inbox product. So, we can just close this PR.
image

**Comment by pftg** [comment]
@sweatC

**Comment by sweatC** [comment]
Waiting for validation for a few days.

**Comment by pftg** [comment]
@sweatC close this pr and create new one and set date

**Pull request closed without merging**